teps/0096-pipelines-v1.md

```diff
@@ -332,6 +332,15 @@ Each V1 CRD should be fully documented before being released, but release of a V
 unrelated documentation issues.
 Once we've achieved the goals of this TEP, we can release a v1 software version of Pipelines.
 
+### Migration plan
+We will start serving v1 versions after the initial release with [v1 CRDs](https://github.com/tektoncd/pipeline/pull/5579).
+And then one release later, we will swap the stored version to v1, mark v1beta1 as deprecated, and document that the EOL
+for v1beta1 is 9 months after that release.
+
+The initial release with v1 CRDs served can serve as a v1 preview, so that we can start migrating the CLI and dashboard, and then
+ v1 will become the official version in the next release. Therefore, supporting v1 versions of CLI, dashboard and any changes to
+ triggers should be prioritized during the current release cycle.
```

Comment:
NIT: indentation is off

Reply:
Do we want to gate the swapping of the stored release version with `v1` support in CLI and dashboard at least?
Chains and operator should also be on the list. Results support can be added later I think, but that's up to the results team.

Reply:
I'd prefer not. Clients shouldn't need to know which version is the stored version, and the earlier we're able to swap the stored version the easier it will be to add new features to v1 (as long as we're not swapping the stored version in the same release as we're serving v1).